### PR TITLE
fix: fix loading of extensions by removing sample-ui-plugin

### PR DESCRIPTION
**Pull request by a-b-r-o-w-n**
## Description

In #4224, the way builtin plugins are loaded changed which breaks when the `extensions.json` file already existed, due to a failure to load the `sample-ui-plugin`. This just removes that from the manifest.

## Task Item

#minor

### CODE REVIEW

**Review by beyackle — APPROVED** [review]

**Review by beyackle — APPROVED** [review]

File: Composer/packages/extension/src/manager/manager.ts [hunk 1]
```diff
@@ -245,12 +245,15 @@ class ExtensionManager {
       const extensionInstallPath = path.dirname(fullPath);
       const packageJson = (await readJson(fullPath)) as PackageJSON;
       const isEnabled = packageJson?.composer && packageJson.composer.enabled !== false;
+      const metadata = getExtensionMetadata(extensionInstallPath, packageJson);
```
Comment (tonyanziano) [inline Composer/packages/extension/src/manager/manager.ts:248]: why was this moved out of the `if` statement if the `else if` does not contain any reference to `metadata`?

Comment (a-b-r-o-w-n) [reply Composer/packages/extension/src/manager/manager.ts:248]: Good question. I had a version where I was just disabling the extension but setting the other metadata fields. I felt like removing the extension is better but forgot to move this back.

Comment (tonyanziano) [reply Composer/packages/extension/src/manager/manager.ts:248]: we can just move it back on some other PR
